### ISSUE TITLE
PipeBlob: tag is not set on blob

### ISSUE THREAD

**Issue by johanburati** [Azure/azure-storage-azcopy#1443]
### Which version of the AzCopy was used? 
```
$ azcopy --version
azcopy version 10.10.0
```

### Which platform are you using? (ex: Windows, Mac, Linux)
Linux

### What command did you run?
```
echo test2 | azcopy cp $BLOBSAS --metadata "tag2=test2" --from-to=PipeBlob
```

### What problem was encountered?
The tag is not set

### How can we reproduce the problem in the simplest way?
When reading from a file,  the tag get set on the blob:
```
azcopy cp file $BLOBSAS --metadata "tag1=test1"
```
When reading from a pipe , the tag does not get set on the blob:
```
echo test2 | azcopy cp $BLOBSAS --metadata "tag2=test2" --from-to=PipeBlob
```

### Have you found a mitigation/solution?
No

**Comment by zezha-msft** [comment]
Fixed in 10.11.

**Comment by johanburati** [comment]
I've tested it, I confirm it works in **azcopy version 10.11.0*
